### ISSUE TITLE
Ability to Disable Next/Your Turn Sounds for GM

### ISSUE THREAD

**Issue by Noggog** [ironmonk88/monks-little-details#197]
Love the mod by the way!   I noticed when playing around with the combat noises, that for the GM these would fire all the time.  In a scenario with 6 goblins vs 2 PCs, the GM will get bombarded with these noises during play.   I would say noises aren't really needed at all for the GM, as the GM is likely the one hitting the next turn button, and doesn't need any audio cue reminders.   It's mostly for players, I would say.

**Comment by ironmonk88** [comment]
![image](https://user-images.githubusercontent.com/75920956/191142495-24e85bdd-efee-49d9-9f34-b700539d19a0.png)

**Comment by Noggog** [comment]
Perhaps my multi-window setup was just being weird, but these settings felt like they were global.   When i modified them as off as the GM, it turned them off for the players, and vice versa.   I'd like to have them be able to be different per player.

video of me flipping between a GM login and a player login, and you can see changing the settings in one messes with the other.
![firefox_wLgRJPm6VE](https://user-images.githubusercontent.com/24981326/191985155-b8f68560-6fac-407f-9ad8-ab9aad0a90e2.gif)

**Comment by Noggog** [comment]
Did some more testing on my end.   First, I checked if Chrome worked better than Firefox (had a few other of my issues clear up that way).  No difference

But I think it might just be due to me misunderstanding of how the setting systems work.   I assumed they were stored per player (GM vs Player1 vs Player2)... but they seem to instead be stored per browser.   So if I tested in two Firefox tabs, they messed with each other, but if I did a Firefox + Chrome, they stayed separate, thanks to the setting caches probably being stored in different places.

TLDR:
I think there's nothing to fix on your end.  It'll work as I intend when it's not me doing testing locally with two tabs

**Comment by ironmonk88** [comment]
Closing this as it sounds like it was a miscommunication.
It's true, client side settings are stored per browser.  Which... I think is not one of the best design choices for Foundry.
I did develop Monk's Player Settings to help with this.  It will try and keep your settings in sync between browsers.
